chart/templates/_helpers.yaml

```
@@ -146,9 +146,13 @@ If release name contains chart name it will be used as a full name.
   securityContext:
     runAsUser: {{ .Values.dags.gitSync.uid }}
   env:
-    {{- if .Values.dags.gitSync.sshKeySecret }}
+    {{- if or .Values.dags.gitSync.sshKeySecret .Values.dags.gitSync.sshKeyCustomFile }}
     - name: GIT_SSH_KEY_FILE
+      {{- if .Values.dags.gitSync.sshKeyCustomFile}}
+      value: {{ .Values.dags.gitSync.sshKeyCustomFile | quote }}
+      {{- else }}
       value: "/etc/git-secret/ssh"
+      {{- end }}
```

```suggestion
      value: {{ .Values.dags.gitSync.sshKeyCustomFile | default "/etc/git-secret/ssh" | quote }}
```

Just noticed this can also be simplified.